releases/note-source/v17_0.rst

```diff
@@ -0,0 +1,299 @@
+
+Proper motion support added to reference catalogs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Reference catalogs now support providing proper motions and parallaxes.
+If proper motions are available, then it will be applied to the specified epoch when the catalog is read.
+No correction for parallax is currently made.
+For more information, refer to `this community.lsst.org post`__ and :jira:`RFC-368`.
+
+__ https://community.lsst.org/t/proper-motion-support-added-to-reference-catalogs/3215
+
+.. _release-v17-0-ingestDriver:
+
+Parallelized image ingest
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The new ``ingestDriver.py`` command makes it possible to ingest images to a data repository in parallel, providing significant speed-ups.
+For more information, refer to `this community.lsst.org post`__.
+
+__ https://community.lsst.org/t/introducing-ingestdriver-py/2599
+
+.. _release-v17-0-runIsr:
+
+Stand-alone instrument signature removal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The new ``runIsr.py`` command makes it possible to run instrument signature removal (ISR) from the command line.
+For more information, refer to `this community.lsst.org post`__.
+
+__ https://community.lsst.org/t/runisr-py-can-now-run-stand-alone-isr-processing-from-the-command-line/3275
+
+.. _release-v17-0-ap:
+
+Alert production capabilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The prototype Alert Production Pipeline is now included in the Science Pipelines software distribution.
+This comprises three new software packages:
+
+- dax_ppdb implements the Prompt Products Database;
+- ap_association implements multiple DIASource detections to compose DIAObjects;
+- ap_pipe provides command-line task to orchestrate the pipeline, running CCD processing, image differencing, and source association.
+
+For more information, refer to `this community.lsst.org post`__ and :jira:`RFC-521`.
+
+__ https://community.lsst.org/t/ap-pipeline-now-in-the-stack/3298
+
+.. _release-v17-0-obs-lsst:
+
+obs_lsst camera package
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This release provides obs_lsst, a new camera description and configuration package which will provide support for data from the LSST Camera, ComCam, the Auxiliary Telescope camera, camera test stand 8, ImSim, and PhoSim.
+This package is expected to be updated and maintained for the indefinite future.
+Previous packages which provide some of the same functionality (e.g. obs_lsstSim) are provided with this release, but will ultimately be deprecated.
+For more information, refer to the documentation for :ref:`lsst.obs.lsst`.
+
+.. _release-v17-0-dcr:
+
+DCR-corrected template construction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This release contains code for generating image differencing templates corrected for the effects of differential chromatic refraction (DCR).
+For information about the technique adopted, refer to :dmtn:`037`; for documentation on the code, see `lsst.ip.diffim.DcrModel`.
+
+.. _release-v17-0-ppmb:
```

Comment:
Wow ppdb and ppmb are close to each other :)

Not asking for a change, I was just confused for a second.